lib/turn.rb

```diff
@@ -0,0 +1,20 @@
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    guess==card.answer
+  end
+
+  def feedback
+    if guess==card.answer
```

Comment:
The `correct?` method in lines 8-10 should be used to determine the outcome of the `feedback` method.